server/services/item.service.ts

```diff
@@ -25,47 +11,30 @@ export const readClusterFromUser = async (username: string) => {
       .member_of({ select: { cluster_id: true } }),
   );
 
-  var clusterSelect = await prisma.cluster.findMany({
-    where: { id: { in: clusterUserList.map((obj) => obj.cluster_id) } },
-    select: {
-      clustername: true,
-      description: true,
-      creator: true,
-    },
+  let count = await prisma.cluster.aggregate({
+    _count: { id: true },
   });
 
-  var cluster = Object.fromEntries(
-    Object.entries(clusterSelect).map(([key, value]) => [
-      key,
-      {
-        type: { category: 0 },
-        tags: null,
-        title: value.clustername,
-        description: value.description,
-        host: value.creator,
-        room: null,
-        link: '/',
+  let result = [];
+  for (let i = 0; i * 5 < count._count.id; i++) {
+    result[i] = await prisma.cluster.findMany({
+      where: { id: { in: clusterUserList.map((obj) => obj.cluster_id) } },
+      select: {
+        id: true,
+        clustername: true,
+        description: true,
+        creator: true,
+        person: { select: { username: true } },
       },
-    ]),
-  );
-  return splitArrayToMultidimensionalArray(cluster, 5) as Item[][];
+      orderBy: { clustername: 'asc' },
+      skip: i * 5,
+      take: i * 5 + 5,
+    });
+  }
+  return result;
```

Comment:
Hmm, da brauche ich eine Erklärung. Warum brauchen wir hier ein for loop? Kann man da nicht mit einer einzigen Abfrage alle Cluster bekommen?